migrator/src/main/scala/com/scylladb/migrator/readers/DynamoDB.scala

```diff
@@ -22,23 +24,32 @@ object DynamoDB {
       if (readThroughput.isEmpty || writeThroughput.isEmpty) Some("25")
       else None
 
-    val df = spark.read
-      .options(
-        (List(
-          source.region.map("region" -> _),
-          source.endpoint.map(ep => "endpoint"                     -> ep.renderEndpoint),
-          source.scanSegments.map(segs => "readPartitions"         -> segs.toString),
-          source.throughputReadPercent.map(pct => "targetCapacity" -> pct.toString),
-          source.maxMapTasks.map(tasks => "defaultParallelism"     -> tasks.toString),
-          throughput.map("throughput" -> _)
-        ).flatten ++ source.credentials.toList.flatMap {
-          case AWSCredentials(accessKey, secretKey) =>
-            List("accesskey" -> accessKey, "secretkey" -> secretKey)
-        }).toMap
-      )
-      .dynamodb(source.table)
+    val jobConf = new JobConf(spark.sparkContext.hadoopConfiguration)
+    def setOptionalConf(name: String, maybeValue: Option[String]): Unit =
```

Comment:
why?
all below are plain strings

why this map and converting?

Reply:
ok, I see
maybe we don't want to duplicate this, is there some util where we can put it so it's not duplicated below?

Reply:
Done in [8e9728d](https://github.com/scylladb/scylla-migrator/pull/105/commits/8e9728d63ff2cdf79f5bb5195b607ba80e684cbe).